src/libproxychains.c

```diff
@@ -479,3 +481,18 @@ struct hostent *gethostbyaddr(const void *addr, socklen_t len, int type) {
 	}
 	return NULL;
 }
+
+ssize_t sendto(int sockfd, const void *buf, size_t len, int flags,
+	       const struct sockaddr *dest_addr, socklen_t addrlen) {
+#ifdef MSG_FASTOPEN
```

Comment:
interesting PR. i assume MSG_FASTOPEN uses the same value across all archs, so imo it would be better to have

```
#ifndef MSG_FASTOPEN
#define MSG_FASTOPEN 0x20000000
#endif
```
and remove the conditional compilation.
that way we produce a .so that works the same way on machines with old and new kernels